zulipterminal/model.py

```diff
@@ -171,6 +179,7 @@ def __init__(self, controller: Any) -> None:
         self.active_emoji_data = OrderedDict(sorted(all_emoji_data,
                                                     key=lambda e: e[0]))
 
+        self._store_content_length_restrictions()
```

Comment:
This is cleaner, though I'm often torn between seeing the higher level details and explicit setting of attributes in `__init__`.